src/install_cmd.js

```diff
@@ -31,24 +32,32 @@ export default function installCmd (cwd, argv) {
   if (argv.debug) logLevel = 'debug'
   if (argv.verbose) logLevel = 'info'
 
+  let progress
+  if (!logLevel) {
+    progress = new ProgressBar(':percent    :current / :total   installing modules', { total: 1 })
+  }
+
   const explicit = !!(argv._.length - 1)
   const updatedPkgJSONs = explicit
     ? entryDep.fromArgv(cwd, argv)
     : entryDep.fromFS(cwd)
 
-  const resolved = updatedPkgJSONs::install.resolveAll(cwd)::skip(1)
+  const resolved = updatedPkgJSONs::install.resolveAll(progress, cwd)::skip(1)
     ::filter(({ local }) => !local)
     ::_do(logResolved.bind(null, logLevel))
     ::publishReplay().refCount()
 
   const linked = resolved::install.linkAll()
-  const fetched = resolved::install.fetchAll(logLevel)
+  const fetched = resolved::install.fetchAll(logLevel, progress)
 
   // only build if we're allowed to.
   const built = argv.build
     ? resolved::install.buildAll()
     : EmptyObservable.create()
 
-  return fsCache.init()
+  const merged = fsCache.init()
     ::concat(linked::merge(fetched)::concat(built))
+
+  merged.subscribe(null, null, () => progress && progress.tick())
```

Comment:
@rase- Can we do this in a `_do` instead of subscribing to the observable? Want to keep obervables cold if possible.